railties/lib/rails/generators/database.rb

```diff
@@ -27,6 +27,38 @@ def gem_for_database(database = options[:database])
         end
       end
 
+      def docker_for_database_build(database = options[:database])
+        case database
+        when "mysql"          then "build-essential default-libmysqlclient-dev git"
+        when "trilogy"        then "build-essential default-libmysqlclient-dev git"
+        when "postgresql"     then "build-essential git libpq-dev"
+        when "sqlite3"        then "build-essential git libvips"
```

Comment:
I'm curious, why `libvips` is included here?

Reply:
Hey Guillermo, thanks for the feedback!  I will remove `libvips` there's no reason.  I just tested it and we are good without it in the `case` statement.

The main reason I'm including `build-essential` and `git` is so that if you switch to another DB type and switch back to `sqlite3` it will implement a proper build step sincer `sqlite3` doesn't have a linux package to install at this point.